frontend/src/Movie/Delete/DeleteMovieModalContent.js

```diff
@@ -24,7 +24,8 @@ class DeleteMovieModalContent extends Component {
 
     this.state = {
       deleteFiles: false,
-      addImportExclusion: false
+      addImportExclusion: false,
+      previousMovie: ''
```

Comment:
not needed in state I don't think, you are only using the value from props and it's never changed in this component